ex2/ex2.c

```diff
@@ -9,6 +9,20 @@
 int main(void)
 {
     // Your code here 
-    
+    FILE *text_txt = fopen("text.txt", "w");
+    int pid = fork();
+
+    if(!pid)
+    {
+        fprintf(text_txt, "Child\n");
+    }
+    else
+    {
+        fprintf(text_txt, "Parent\n");
+    }
     return 0;
 }
+
```

Comment:
For future reference get in the habit of closing files when you're finished with fclose()